resources/views/frontend/catalog-new/index-new.blade.php

```diff
@@ -57,16 +57,16 @@ class="tw-border-2 tw-border-gray-800 tw-bg-white tw-px-8 tw-py-16">
                                     <span>year range</span>
                                 </template>
                                 <template #body>
-                                    <div
-                                        class="tw-absolute tw-left-0 tw-top-36 tw-z-10 tw-w-screen tw-px-16">
-                                        <div class="tw-border-2 tw-border-gray-800 tw-bg-white tw-p-6">
+                                    <div class="tw-absolute tw-top-36 tw-z-10">
+                                        <div
+                                            class="tw-w-[27.5rem] tw-border-2 tw-border-gray-800 tw-bg-white tw-p-6 tw-pt-4">
```

Comment:
the `tw-w-[27.5rem]` seems strange, round it up to 28? :)
```suggestion
                                            class="tw-w-[28rem] tw-border-2 tw-border-gray-800 tw-bg-white tw-p-6 tw-pt-4">
```